packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -188,6 +191,7 @@ export default function makeKernelKeeper(storage) {
       case undefined:
         throw new Error(`unknown kernelPromise '${kernelSlot}'`);
       case 'unresolved':
+        p.refCount = storage.get(`${kernelSlot}.refCount`);
```

Comment:
Might be safer to do the `Nat(Number(` here, and keep an integer in `p.refCount` instead of a string.